lib/common/resources/neuron/pytorch-inference-resnet50.yml

```diff
@@ -0,0 +1,96 @@
+apiVersion: v1
```

Comment:
This file should be moved to aws-observability-accelerator-repo and i would prefer using a GitOps approach to deploy the workload

Reply:
it is an example workload, just generating metrics, not a real one never to be used in production.
so I believe deployed manually as a pod is better?

Reply:
I would say deploy this manually as a pattern. DockerFile should be moved to a different repo may be `aws-observability-accelerator-repo `